2021/zrimsek2022learning/openreview/content.tex

```diff
@@ -0,0 +1,260 @@
+We can also see that prediction of PIPR is slower, but that is probably because of model using default predict batch size, which is 32 (GNN-PPI uses 256) and could be improved. Even if not, we have around 500 predictions per second, so prediction time shouldn't be a problem.
+
+\begin{table}[h] \caption{\label{times}\textbf{Training and prediction times.}}\centering
+\begin{tabular}{ccccccc} 
+    \toprule
+    \multirow{2}{*}[-3pt]{} & \multicolumn{3}{c}{Training time [min]} & \multicolumn{3}{c}{Testing time [s]} \\ 
+        \cmidrule(l{5pt}r{5pt}){2-4} \cmidrule(l{5pt}r{5pt}){5-7} 
+      & SHS27k & SHS148k & STRING  & SHS27k & SHS148k & STRING \\ 
+         \midrule
+    GNN-PPI & 11 & 63 & 1064 & 0.2 & 1.2 & 44 \\ 
+
+    PIPR & 36 & 135 & 602 & 5 & 23 & 291 \\ 
+    \bottomrule
+\end{tabular}
+\end{table}
+
+\section{Results}
+\label{sec:results}
+In this section we will present the results of the experiments that we did to support the claims above. Our results were in some parts far from the authors results, but even so, they support the main claims of the original paper.
+
+\subsection{Comparison of GNN-PPI and PIPR}
+Results in this subsection refer to our first two claims. In Table \ref{subsets} we can see the comparison of the models in question on all three datasets, trained and tested with all three partition schemes.
+For first two datasets we ran the experiments 5 times, with 5 different random seeds, to see how the performance changes on different sets. We can see that especially for SHS27k the standard deviation for \emph{bfs} and \emph{dfs} splits is very big, which also explain why our results are at some points quite far from the authors. For STRING, multiple runs were too expensive, so we only trained and tested once. But this should be enough to asses the authors claims, because the dataset is so much larger, that the randomness of the split effects the performance less.
+
+The comparison of general performance, that refers to the first claim, can be seen if we look at the micro-F1 score averaged across whole dataset (column \emph{average}). Except for BFS mode on the smallest dataset, where uncertainty is too big, we \textbf{can confirm} that in all other cases \textbf{GNN-PPI performs better}.
+
+The second claim is that GNN-PPI predicts inter-novel-protein interactions better. The subsets we are observing are denoted with \emph{bs}, standing for both proteins being seen during training, \emph{es}, either of the proteins seen during training and \emph{ns}, neither of the proteins seen before. So for the second claim, we need to compare the performance on subsets \emph{es} and \emph{ns}. The results in the table are \textbf{in bold where} uncertainty is small enough that \textbf{we can confirm it}. We can also observe that even where it is not, mean is never bigger for PIPR. 
+
+\begin{table}[h]
```

Comment:
The compilation is breaking only due to this table. If you remove this table the compilation works fine. Some things you can try to fix the table:

- Have all underscores within math mode, i.e. `X_{BS}` --> `$X_{BS}`
- Have all `\pm` enclosed by math mode, i.e. `\pm 10` --> `$\pm 10$`

Reply:
Thanks for your answer. I had newly defined command for type of cell so that I wouldn't need to enclose everything in math mode and I guess it didn't work here. I changed that now, but it is still not compiling for me - it also doesn't work if I remove the table. Is it maybe working for you now, since it did before without the table?